spec/system/composer/drafts_spec.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+describe "Composer - Drafts", type: :system do
+  fab!(:topic)
+  fab!(:current_user, :admin)
+
+  let(:toasts) { PageObjects::Components::Toasts.new }
+  let(:topic_page) { PageObjects::Pages::Topic.new }
+  let(:composer) { PageObjects::Components::Composer.new }
+  let(:discard_draft_modal) { PageObjects::Modals::DiscardDraft.new }
+
+  before { sign_in(current_user) }
+
+  context "when clicking X (save and close)" do
+    it "saves the draft and shows a toast" do
+      visit "/new-topic"
+
+      expect(composer).to be_opened
+      composer.fill_title("this is a test topic")
+      composer.fill_content("a b c d e f g")
```

Comment:
So meta? It's a composer, and these are all the (non-flat) notes.

Reply:
:joy: I'll have to trust you on that I know nothing about reading music or music theory haha